libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/GoogleFragment.java

```diff
@@ -246,4 +249,13 @@ public void onActivityResult(int request, int result, Intent data) {
                 break;
         }
     }
+
+    // Remove scale from photo URL path string. Current URL matches /s96-c, which returns a 96 x 96
+    // pixel image. Removing /s96-c from the string returns a 512 x 512 pixel image. Using regular
+    // expressions may help if the photo URL scale value in the returned path changes.
+    protected String removeScaleFromGooglePhotoUrl(String photoUrl) {
```

Comment:
The reason I didn't was because I wanted to take a look at it later on, to check if it's actually working or even if it's still needed. Maybe we should keep track of these smaller improvements in a list so we can tackle them in separate PRs, what do you say?

Reply:
Sounds like a good plan. If we ever come back to this class, I think we'll find more stuff like this.